src/cli/mod.rs

```diff
@@ -195,4 +239,22 @@ mod tests {
 
         parse_args(args).unwrap();
     }
+
+    #[test]
+    fn test_exp_cmd() {
+        let args: Vec<_> = [
+            "afterburn",
+            "exp",
+            "rd-network-kargs",
+            "--provider",
+            "gcp",
+            "--default-value",
+            "ip=dhcp",
+        ]
+        .iter()
+        .map(ToString::to_string)
+        .collect();
+
+        parse_args(args).unwrap();
```

Comment:
Shouldn't we check here that the parsed args correspond to the right enum variant and subvariant?